README.md

```diff
@@ -43,23 +48,48 @@ port          | 8080                               | Port-number to run the Hood
 bindAddress   | 127.0.0.1                          | Address that Hoodie binds to
 public        | path.join(options.path, 'public')  | path to static assets
 inMemory      | false                              | Whether to start the PouchDB Server in memory
-dbUrl         | PouchDB Server                     | If provided does not start PouchDB Server and uses external CouchDB. Has to contain credentials.
+dbUrl         | –                                  | If provided uses external CouchDB. Has to contain credentials.
 data          | path.join(options.path, '.hoodie') | Data path
 
-If that doesn’t make much sense just yet, don’t worry about it.
+Hoodie is using the [rc](https://www.npmjs.com/package/rc) module to retrieve
+configuration from CLI arguments, environment variables and configuration files.
 
 ## Testing
 
-The `@hoodie/server` test suite is run with `npm test`.
+Local setup
+
+```
+git clone https://github.com/hoodiehq/hoodie.git
+cd hoodie
+npm install
+```
+
+The `hoodie` test suite is run with `npm test`.
+
+You can [read more about testing Hoodie](test)
```

Comment:
`(test)` links to the test folder, see https://github.com/hoodiehq/hoodie-server/blob/camp/README.md